### DISCUSSION TITLE
manually sortable tags

### DISCUSSION THREAD

**Discussion by angobango** [standardnotes/forum#2546] [Suggestions]
hello
is there a chance to get tags sortable manually, instead of forced alphabetical order? i use standardnotes to prepare texts for websites and i'd like to sort tags as they had the same hierarchical structure of the navigation, including individually sortable 'sub-tags' along with the folder-extension. 
or am i missing something?

thank you in advance!

regards
angelo

**Comment by holzi-x** [comment]
I guess you could get something "handknitted" using smart tags. see here:
[https://standardnotes.org/help/42/how-do-i-view-a-list-of-untagged-notes](https://standardnotes.org/help/42/how-do-i-view-a-list-of-untagged-notes)

**Comment by angobango** [comment]
thank you for your reply. i looked over it, but it seems to be more useful for filtering purposes. i simply updated my tags with leading numbers now, so it has the desired structure. thanks anyway!

**Comment by jrabensc** [comment]
I would appreciate this as well. Is there any particular reason for it currently missing?

**Comment by cgw4455235** [comment]
bump

**Comment by arielsvg** [comment]
Many desirable features aren't there simply because we haven't gotten around to implementing them yet 🙂 building software takes time

**Comment by mikeaarts** [comment]
I’d also like to vote for arbitrary tag sequencing please. But definitely appreciate that good software takes time, so thanks for taking the time and care to build a great product.
